### ISSUE TITLE
Request for releasing a official binary version for Arch User Repository (AUR) 

### ISSUE THREAD

**Issue by tasnuva1** [pieces-app/pieces-os-client-sdk-for-typescript#15]
- `Non-systemd` users don't want to install Snan in their custom build since it is bloated.
- It is necessary to install form AUR helper like `yay`,
- At this point, it's a dying request. I hope that you'll consider publishing the official binary version of pieces-for-developers-bin.

**Comment by jordan-pieces** [comment]
@tasnuva1 can you elaborate a little more on this issue? would love to make this available for you (and other's) in the most ideal way possible 👍 

I will start some other conversations to get information on what you have shared so far

**Comment by tasnuva1** [comment]
- For Linux installation, You have to install [snap](https://wiki.archlinux.org/title/Snap) - a package manager, and a bloated system for some custom-built Linux.
- For `non-systemd` users like myself, specifically, [Artix Linux](https://artixlinux.org/) which uses simple, fast `init systems`. ( I use Podman an alternative to Docker if needed, just a sight note if you are curious)
- For Arch Linux users, it has a community-driven repository called the [AUR](https://wiki.archlinux.org/title/Arch_User_Repository) which is home to thousands of packages contributed by Arch users.
- An AUR helper like [Yay](https://aur.archlinux.org/packages/yay) can install packages from the Arch User Repository (AUR).
- In short, It's a dying need and a request for an official binary version, targeting Arch users.

**Comment by jordan-pieces** [comment]
@nathan-courtney-pieces i wanted to see if you could be of any assistance here around specifics as the Linux Maestro 👍

**Comment by nathan-courtney-pieces** [comment]
Currently we only support snap. I will be looking into distributing to Flatpak sometime in the future

**Comment by tasnuva1** [comment]
> Flatpak

Is there any change to a Linux 64-bit AppImage?

**Comment by jordan-pieces** [comment]
@nathan-courtney-pieces just wanted to bump this here for @tasnuva1 👍
